presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -499,6 +516,10 @@ private void flushColumnWriters(FlushReason flushReason)
             }
         }
 
+        if (dwrfStripeCacheWriter.isPresent()) {
```

Comment:
use the same ifPresent pattern like footer.

Reply:
Lambdas require all the vars to be final or effectively final, unfortunately it doesn't work for `indexLength`.